src/main/java/io/vinyldns/java/VinylDNSClientImpl.java

```diff
@@ -335,11 +344,56 @@ public VinylDNSClientImpl(VinylDNSClientConfig config) {
   @Override
   public VinylDNSResponse<BatchResponse> createBatchChanges(CreateBatchRequest request) {
     String path = "zones/batchrecordchanges";
+
+    VinylDNSRequest<CreateBatchRequest> vinylDNSRequest =
+        new VinylDNSRequest<>(Methods.POST.name(), getBaseUrl(), path, request);
+
+    if (request.getAllowManualReview() != null) {
+      vinylDNSRequest.addParameter("startFrom", request.getAllowManualReview().toString());
```

Comment:
should be `allowManualReview` not `startFrom`

Reply:
Oof, nice catch